grading/ccorrect/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# DOCKER-VERSION 1.1.0
+
+#inherit from the base container
+ARG   VERSION=latest
+FROM  ingi/inginious-c-base:${VERSION}
+LABEL org.inginious.grading.name="ccorrect"
+
+# install gcc-toolset-12-gdb, libasan, libtsan, debug symbols for some libraries and install CCorrect
+RUN     dnf config-manager --set-enabled powertools && \
+        dnf -y install gcc make git cppcheck gcc-toolset-12-gdb libasan libtsan && \
+        dnf -y --enablerepo=baseos-debug debuginfo-install libgcc glibc libasan libtsan libstdc++ && \
+        dnf clean all && \
+        pip3 install git+https://github.com/mpostaire/CCorrect.git --target=/usr/lib/python3.6/site-packages/
```

Comment:
Unless you explicitly need python 3.6 for your tool, I recommend to keep python3.8 as the inginious API is only copied in those folders.

Reply:
CCorrect can be used from both inside and outside gdb and gdb 11.2's python interpreter is version 3.6.

I edited the Dockerfile so that CCorrect is installed normally and the PYTHONPATH is prepended to add the directories where its dependencies and itself are installed. That way can access CCorrect as it reads PYTHONPATH.